test/sdk.test.ts

```diff
@@ -93,6 +97,20 @@ describe('Tests the SDK functionality', () => {
 
         expect(autoPerformanceIsEnabled()).toBe(true);
       });
+
+      it('Do not overwrite user defined integrations', () => {
+        const nav = new ReactNavigationInstrumentation();
+        const tracing = new ReactNativeTracing({ routingInstrumentation: nav });
+        init({
+          tracesSampleRate: 0.5,
+          enableAutoPerformanceTracking: true,
+          integrations: [tracing],
+        });
```

Comment:
Do you think it makes sense to test a similar scenario where `defaultIntegrations: [tracing]` is specified in the init options? Just to make sure that in this case, `tracing` isn't overridden either.

Reply:
Yep, makes sense, Done.